### PR TITLE
Fixed bugs of using report portal client on windows

### PR DESCRIPTION
**Pull request by PositiveAlex**
Having a bug with join paths on windows (windows use backslash \ as path separartor, not forward slash /)
Avoided using platform dependent `os.path.join` to work similar on Windows as on Linux. 

Can someone test this PR on Linux before merge?

### CODE REVIEW

**Comment by PositiveAlex** [comment]
@avarabyeu Andrey, could you please forward this to someone to make a review?

File: reportportal_client/service.py [hunk 1]
```diff
@@ -6,50 +6,75 @@
...
+        self.base_url = self.uri_join_safe(self.endpoint,
+                                           self.api_base,
+                                           self.project)
         self.headers = {"Content-Type": "application/json",
                         "Authorization": "{0} {1}".format("bearer",
                                                           self.token)}
         self.session = requests.Session()
 
+    @staticmethod
+    def uri_join_safe(*uri_parts):
```
Comment (avarabyeu) [inline reportportal_client/service.py]: i would rename this method. 'safe' doesn't means nothing and does not give an idea to reader what this method actually does. In general commit is OK.

**Comment by avarabyeu** [comment]
@tkachoff any concerns?

File: reportportal_client/service.py (hunk 1 above)
Comment (PositiveAlex) [inline reportportal_client/service.py]: @avarabyeu fixed

**Comment by PositiveAlex** [comment]
Thank you, I'm awaiting for merge

**Pull request merged**